package.json

```diff
@@ -33,6 +33,7 @@
     "import-google-fonts": "^0.0.2",
     "js-cookie": "^3.0.1",
     "moment": "^2.29.4",
+    "mysql2": "^3.9.2",
```

Comment:
Can be removed through `npm uninstall mysql2` followed by `npm install` (to update the package-lock) once you switch over to using prisma